Tools/CISupport/ews-build/config.json

```diff
@@ -392,6 +392,11 @@
       "factory": "UnsafeMergeQueueFactory", "platform": "mac-monterey",
       "configuration": "release", "architectures": ["arm64"],
       "workernames": ["webkit-cq-01", "webkit-cq-02", "webkit-cq-03", "webkit-cq-04", "webkit-cq-05"]
+    },
+    {
+      "name": "Safe-Merge-Queue", "shortname": "safe-merge", "icon": "testOnly",
+      "factory": "SafeMergeQueueFactory", "platform": "*",
+      "workernames": ["ews151"]
```

Comment:
Presumably this is just for testing, the final version should be run on the same machines tasked with `merge-queue`.